src/validationinterface.cpp

```diff
@@ -179,8 +179,8 @@ void CMainSignals::BlockChecked(const CBlock& block, const CValidationState& sta
     m_internals->BlockChecked(block, state);
 }
 
-void CMainSignals::NewPoWValidBlock(const CBlockIndex *pindex, const std::shared_ptr<const CBlock> &block) {
-    m_internals->NewPoWValidBlock(pindex, block);
+void CMainSignals::NewPoSValidBlock(const CBlockIndex *pindex, const std::shared_ptr<const CBlock> &block) {
+    m_internals->NewPoSValidBlock(pindex, block);
```

Comment:
Maybe it would make sense to reduce the surface area of this pull request to separate the renaming of this function out into a separate pull request. I think it was missed and should have been done when activating PoS and removing PoW already.

Reply:
I would need to rename branch then =(

Reply:
Why does this require renaming any branch? Do we have a branch-naming scheme and a corporate policy that leads to ... bad things when not followed that I am not aware of?

Reply:
We don't have such rule of course. Just not sure this rename is so important. But I will do it in a separate PR if you wish =)

Reply:
Take it as a Nit. Do as you wish. This thing is perfectly reviewable with it in it. Just that reflex sprung up again.